addons/website/static/src/builder/plugins/theme/theme_colors_option.xml

```diff
@@ -68,14 +68,14 @@
                     </div>
                 </div>
             </BuilderRow>
-            <BuilderRow label.translate="Color Presets">
+            <BuilderRow label.translate="Color Presets" expand="colorPresetToShow">
                 <div></div>     <!-- This is required, without it the row is not displayed at all -->
                 <t t-set-slot="collapse">
                     <t t-foreach="state.presets" t-as="preset" t-key="preset.id">
-                        <BuilderRow t-slot-scope="row">
-                            <div class="o_cc_preview_wrapper" t-on-click="row.toggleCollapseContent">
+                        <BuilderRow t-slot-scope="row" expand="preset.id === colorPresetToShow">
+                            <div class="o_cc_preview_wrapper w-100" t-on-click="row.toggleCollapseContent">
                                 <div inert="" t-att-class="`o_cc${preset.id}`"
-                                     class="w-100 p-2 d-flex justify-content-between align-items-center ms-3">
+                                     class="p-2 d-flex justify-content-between align-items-center ms-3">
```

Comment:
Small fix: I moved the `w-100` class to the parent div so the color preset takes the full width.